### PR TITLE
Update awesome fonts to 3.0

### PR DESCRIPTION
**Pull request by hdd**
Hi I've updated the fonts to 3.0.2 .
Cheers.
L.

### CODE REVIEW

**Comment by martinpengellyphillips** [comment]
Thanks for this, but I think you also need to update font_awesome.styl to include the definitions for the new icons.

**Comment by hdd** [comment]
I'll add it and push the changes.

File: source/dbootstrap/theme/dbootstrap/font_awesome.styl [hunk 1]
```diff
@@ -100,6 +100,8 @@ ul.icons {
 
 // Font Awesome uses the Unicode Private Use Area (PUA) to ensure screen
 // readers do not read off random characters that represent icons
+/*  Font Awesome uses the Unicode Private Use Area (PUA) to ensure screen
```
Comment (martinpengellyphillips) [inline source/dbootstrap/theme/dbootstrap/font_awesome.styl]: Please remove these duplicate lines.

File: source/dbootstrap/theme/dbootstrap/font_awesome.styl [hunk 2]
```diff
@@ -132,7 +134,7 @@ ul.icons {
 .icon-play-circle:before          { content: "\f01d"; }
 .icon-repeat:before               { content: "\f01e"; }
 
-// \f020 doesn't work in Safari. all shifted one down
+/* \f020 doesn't work in Safari. all shifted one down */
```
Comment (martinpengellyphillips) [inline source/dbootstrap/theme/dbootstrap/font_awesome.styl]: Please use // for comments rather than /\* */ pairs.

**Comment by martinpengellyphillips** [comment]
Looking good. Any chance you can add in the icon-spin and icon-border classes as well?

**Comment by hdd** [comment]
Looking now into the missing bits.

**Comment by hdd** [comment]
It probably worth having some custom icon example in the gallery to show these new icons and how they work.

**Comment by martinpengellyphillips** [comment]
Hey. I have submitted a pull request to your branch with your proposed changes (add a section showing the modifiers) as well as a few related fixes. If you accept that pull request it should in turn update this one and I can proceed with the merge.

**Comment by hdd** [comment]
Changes are now been merged to my branch and pushed to font-awesome-3.0 branch.

**Pull request merged**